client-app/ui-kit/components/molecules/file/vc-file.vue

```diff
@@ -155,7 +155,7 @@ const fileSize = computed(() => getFileSize(props.file.size));
   }
 
   &__link {
-    @apply text-[--link-color] cursor-pointer;
+    @apply text-[--link-color] cursor-pointer break-all;
```

Comment:
I was need to [made same changes](https://github.com/VirtoCommerce/vc-theme-b2b-vue/commit/1b0cfbf1043a63d6add04eccd2332daa24107c41#diff-98933ad124029fb87a3b3c586eb6febec7505f8d9bd4dee19f180c82c7f193b7).

I think the better solution is to replace `break-words` below ([167 line](https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/db300b020aa81edee39733a9d9f7404c64b7533e/client-app/ui-kit/components/molecules/file/vc-file.vue#L167)) and add use `self-center` instead of `self-start` on `__buttons`